x-pack/plugins/maps/public/connected_components/mb_map/draw_control/draw_control.tsx

```diff
@@ -13,99 +13,130 @@ import MapboxDraw from '@mapbox/mapbox-gl-draw';
 import DrawRectangle from 'mapbox-gl-draw-rectangle-mode';
 import type { Map as MbMap } from '@kbn/mapbox-gl';
 import { Feature } from 'geojson';
-import { DRAW_TYPE } from '../../../../common/constants';
+import { DRAW_SHAPE } from '../../../../common/constants';
 import { DrawCircle } from './draw_circle';
 import { DrawTooltip } from './draw_tooltip';
 
+const SIMPLE_SELECT = 'simple_select';
 const DRAW_RECTANGLE = 'draw_rectangle';
 const DRAW_CIRCLE = 'draw_circle';
+const DRAW_POLYGON = 'draw_polygon';
+const DRAW_LINE = 'draw_line_string';
+const DRAW_POINT = 'draw_point';
+
+const mbModeEquivalencies = new Map([
+  ['simple_select', 'SIMPLE_SELECT'],
+  ['draw_rectangle', 'BOUNDS'],
+  ['draw_circle', 'DISTANCE'],
+  ['draw_polygon', 'POLYGON'],
+  ['draw_line_string', 'LINE'],
+  ['draw_point', 'POINT'],
+]);
 
 const mbDrawModes = MapboxDraw.modes;
 mbDrawModes[DRAW_RECTANGLE] = DrawRectangle;
 mbDrawModes[DRAW_CIRCLE] = DrawCircle;
 
 export interface Props {
-  drawType?: DRAW_TYPE;
-  onDraw: (event: { features: Feature[] }) => void;
+  drawShape?: DRAW_SHAPE;
+  onDraw: (event: { features: Feature[] }, drawControl?: MapboxDraw) => void;
   mbMap: MbMap;
+  drawActive: boolean;
```

Comment:
Nit: "drawActive" does not provide a description of what the prop does. How about renaming to "show" or "enable"?